scripts/json-query.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable max-len */
+export default function query(select, from, where, orderBy, rowCount) {
+  // Validate input parameters
+  if (!Array.isArray(select) || !Array.isArray(from) || !Array.isArray(where) || where.length !== 3) {
+    throw new Error('Invalid input parameters');
+  }
+
+  // Extract the filter condition components
+  const [field, operator, value] = where;
+
+  // Sort the 'from' array based on the orderBy parameter
+  if (orderBy) {
+    const [orderByField, sortOrder] = orderBy;
+    from.sort((a, b) => {
+      const aValue = a[orderByField];
+      const bValue = b[orderByField];
+      if (sortOrder === 'asc') {
```

Comment:
Conditions can be simplified

```suggestion
      if (sortOrder === 'asc') {
  Math.sign(aValue - bValue)
} else if (sortOrder === 'desc') {
   Math.sign(bValue - aValue)
}
```